techniques/flash/FLASH21.html

```diff
@@ -21,7 +21,7 @@ <h3>A statistical data table</h3>
          
             <p>In this example, statistical data is used as data provider for a   dynamically created DataGrid component. The lines import   fl.accessibility.DataGridAccImpl;   DataGridAccImpl.enableAccessibility(); are required to enable   accessibility for the Datagrid Component. </p>
          
-         <pre xml:space="preserve">import fl.accessibility.DataGridAccImpl;
+         <pre>import fl.accessibility.DataGridAccImpl;
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`